### ISSUE TITLE
Error in !opts$quiet : invalid argument type

### ISSUE THREAD

**Issue by kimofos** [MatMatt/MODIS#26]
Hello,
getHdf, runGdal and runMrt give the message above: Error in !opts$quiet : invalid argument type
is there a fix for that?
thanks

**Comment by fdetsch** [comment]
Hi and thanks for letting us know. Unfortunately, I could replicate this behavior neither on Ubuntu Linux nor Windows 10 using the following code:
```{r}
library(MODIS)
MODISoptions() # displays the current default setting for 'quiet', which should either be TRUE or FALSE

hdf <- getHdf("MOD11A2", begin = "2017001", end = "2017015", tileH = 21, tileV = 9)
tfs <- runGdal("MOD11A2", begin = "2017001", end = "2017010", tileH = 21, tileV = 9)
```
Could you please provide us with the output of `sessionInfo()`? For me, it's 

```{r}
> sessionInfo()
R version 3.4.1 (2017-06-30)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                           LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] shiny_1.0.3   MODIS_1.1.0   raster_2.5-8  sp_1.2-5      mapdata_2.2-6 maps_3.2.0   

loaded via a namespace (and not attached):
 [1] leaflet.extras_0.2 sf_0.5-2           lattice_0.20-35    mapview_2.1.0      colorspace_1.3-2   miniUI_0.1.1      
 [7] htmltools_0.3.6    stats4_3.4.1       viridisLite_0.2.0  yaml_2.1.14        base64enc_0.1-3    XML_3.98-1.9      
[13] R.oo_1.21.0        nloptr_1.0.4       withr_1.0.2        DBI_0.7            R.utils_2.5.0      foreach_1.4.3     
[19] plyr_1.8.4         rgeos_0.3-23       munsell_0.4.3      mapedit_0.3.2      R.methodsS3_1.7.1  devtools_1.13.2   
[25] htmlwidgets_0.9    codetools_0.2-15   memoise_1.1.0      httpuv_1.3.5       crosstalk_1.0.0    parallel_3.4.1    
[31] gdalUtils_2.0.1.7  Rcpp_0.12.12       xtable_1.8-2       udunits2_0.13      scales_0.4.1       satellite_0.2.0   
[37] leaflet_1.1.0      webshot_0.4.1      jsonlite_1.5       mime_0.5           ptw_1.9-12         png_0.1-7         
[43] digest_0.6.12      grid_3.4.1         rgdal_1.2-8        tools_3.4.1        bitops_1.0-6       magrittr_1.5      
[49] RCurl_1.95-4.8     iterators_1.0.8    R6_2.2.2           units_0.4-5        compiler_3.4.1   
```
after **MODIS** has been loaded.

**Comment by kimofos** [comment]
Thanks @fdetsch for the fast reply

I tried your example and it worked, but runMrt still doesn't work:

    library(MODIS)
    MODISoptions()

    STORAGE:
    localArcPath : /Volumes/MODIS_EVI/1_HDFs 
    outDirPath   : /Volumes/MODIS_EVI/2_Preprocessed 
    DOWNLOAD:
    MODISserverOrder : LAADS, LPDAAC 
    dlmethod         : auto 
    stubbornness     : high 
    wait             : 0.5 
    quiet            : FALSE 
    PROCESSING:
    GDAL           : GDAL 2.1.3, released 2017/20/01 
    MRT            : Enabled 
    pixelSize      : asIn 
   outProj        : asIn 
   resamplingType : NN 
   dataFormat     : GTiff 

The example you suggested works:

    runGdal("MOD11A2", begin = "2017001", end = "2017010", tileH = 21, tileV = 9)

However, runMrt  fails:

    runMrt( job="test2", product="MOD11A2", tileH=12, tileV=11, begin="2017001",  SDSstring="011000000010",  outProj="GEO", datum="WGS84")

Fehler in !opts$quiet : ungültiger Argumenttyp

I have to use runMrt, as this is update of data that has been processed using runMrt in the past. runGdal produces different results.

thanks!

**Comment by kimofos** [comment]
forgot to add the session info

R version 3.4.1 (2017-06-30)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Sierra 10.12.3

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.4/Resources/lib/libRlapack.dylib

locale:
[1] de_DE.UTF-8/de_DE.UTF-8/de_DE.UTF-8/C/de_DE.UTF-8/de_DE.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] MODIS_1.1.0   raster_2.5-8  sp_1.2-5      mapdata_2.2-6 maps_3.2.0   

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.12    magrittr_1.5    units_0.4-5     devtools_1.13.3 xtable_1.8-2    lattice_0.20-35 R6_2.2.2        ptw_1.9-12      httr_1.2.1      udunits2_0.13  
[11] tools_3.4.1     parallel_3.4.1  rgdal_1.2-8     grid_3.4.1      DBI_0.7         git2r_0.19.0    withr_2.0.0     htmltools_0.3.6 mapedit_0.3.2   digest_0.6.12  
[21] sf_0.5-3        shiny_1.0.3     nloptr_1.0.4    bitops_1.0-6    curl_2.8.1      RCurl_1.95-4.8  memoise_1.1.0   mime_0.5        compiler_3.4.1  XML_3.98-1.9   
[31] httpuv_1.3.5

**Comment by rokoeh** [comment]
I had this trouble. When I set quiet=T it works!

**Comment by Maria-da-Conceicao-Pereira** [comment]
I also encountered this problem. Like @rokoeh I added the argument 'quiet' from MODISoptions, set it to TRUE, and managed to execute the download I wanted (function getHdf). For example:
`# Year 2014
transDate(begin = "2014-01-01", end = "2014-12-30")
getHdf(product = "MCD64A1", begin = "2014-01-01", end = "2014-12-30", quiet = TRUE)`
Greetings

**Comment by fdetsch** [comment]
Fixed as of c6d49887321c97c647e5ce7196f165c95f7ecbd8.
